server/api/unit/unit.model.ts

```diff
@@ -30,7 +30,7 @@ export class Unit extends BaseEntity {
   @Column({
     default: true,
   })
-  includeDefaultConfig: boolean;
+  includeDefaultConfig!: boolean;
```

Comment:
I'll have to pick your brain on this. A boolean is already required, right? Why the `!`?

Reply:
This is to fix a compiler error that gets thrown if you have a non-optional member variable defined with no default value and no value set in the constructor. It's just telling the compiler you know what you're doing. For some reason we were getting away with this one, I guess because we hadn't touched that file in a long time or something.